src/Concordium/Client/Commands.hs

```diff
@@ -967,6 +1058,7 @@ accountUpdateCredentialsCmd =
                 <$> optional (strOption (long "new-credentials" <> metavar "FILE" <> help "File containing the new credential deployment informations."))
                 <*> optional (strOption (long "remove-credentials" <> metavar "FILE" <> help "File containing credential registration ids of the credentials to be removed."))
                 <*> option auto (long "new-threshold" <> metavar "THRESHOLD" <> help "New account threshold, i.e. how many credential holders needed to sign a transaction.")
+                <*> optional (strOption (long "outFile" <> metavar "FILE" <> help "An optional file name and path to ouptput the signed/partially-signed transaction to instead of submitting the transaction on-chain."))
```

Comment:
I think this is not needed since it is already in `transactionOptsParserBuilder` used in `transactionOptsParser` :)